README.md

```diff
@@ -101,7 +102,7 @@ City Union | :cn: Mainland China | :new: `IOS`
 [Go-to card][mspgoto] | :us: Minneapolis-St. Paul, MN, USA | :new: :closed_lock_with_key: `MFC`
 [Hafilat][] | :united_arab_emirates: Abu Dhabi, UAE | :new: `IOS`
 [Hop Fastpass][hfp] | :us: Portland, OR, USA | :new: :id: `IOS`
-[HSL][hsl], [Matkakortti][matka] | :finland: Finland | `IOS`
+[HSL][hsl], [Matkakortti][matka], [Waltti][] | :finland: Finland | `IOS`
```

Comment:
Missing reference

Reply:
It appears to render correctly here: https://github.com/metrodroid/metrodroid/blob/36d4569731224d919ce6fe74c562dade810a5373/README.md

We have another spot with a case-mismatching reference, `Selecta`.